prow/hook/events.go

```diff
@@ -77,9 +80,13 @@ func (s *Server) handleReviewEvent(l *logrus.Entry, re github.ReviewEvent) {
 				re.Repo.Name,
 				re.PullRequest.Number,
 			)
+			start := time.Now()
+			labels := prometheus.Labels{"event_type": l.Data["event-type"].(string), "action": string(re.Action), "plugin": p}
```

Comment:
Forgive me for the sin ... but threading through the event type data would be challenging otherwise

Reply:
Can we then at least make the key a const?